frontend/src/Organize/OrganizePreviewModalContent.js

```diff
@@ -93,7 +96,7 @@ class OrganizePreviewModalContent extends Component {
     return (
       <ModalContent onModalClose={onModalClose}>
         <ModalHeader>
-          Organize & Rename {seasonNumber != null && <SeasonNumber seasonNumber={seasonNumber} />}
+          {translate('OrganizeModalHeader', { season: seasonNumber != null && ReactDOMServer.renderToString(<SeasonNumber seasonNumber={seasonNumber} />) || '' })}
```

Comment:
I don't think we should be using `ReactDOMServer` on the client side, but given how simple the `SeasonNumber` component is we can just replace it with a function. Looks like we use it in a few spots, but use it the same way in all.

Reply:
replaced the SeasonNumber component with a seasonFormat function. wasn't too sure on the name of it!